salesagility/SuiteCRM
# add missing icon from Document Revisions sub panel

## Description
Add an icon for document revisions subpanel to Suite P.
## Motivation and Context
Icon was missing (task 250 on kanboard)

## How To Test This
Set your profile to use the Suite P them then go to details page of a document. If the fix has worked there should be an icon (pencil on a square) on the document revisions sub panel. Before the fix this subpanel was the only one that didn't have an icon before it's title.

## Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)

### Final checklist
